Resources/NetHookAnalyzer2/NetHookAnalyzer2/Generated/.gitattributes

```diff
@@ -0,0 +1 @@
+*.cs text eol=crlf -diff
```

Comment:
why not let autocrlf take care of it?

Reply:
It was mostly for -diff, and I copied it from steamkits generated folder.

Reply:
why not diff some nice text files?

FWIW GitHub seems to be ignoring it anyway

Reply:
I removed it.